tasks/main.yml

```diff
@@ -1,35 +1,52 @@
 ---
 - name: Install Nokogiri requirements
-  apt: pkg={{ item }} state=latest
-  with_items:
-    - libxslt1-dev
-    - libxml2-dev
+  apt:
+    pkg:
+      - libxslt1-dev
+      - libxml2-dev
+    state: latest
 
 - name: Install Ruby FFI
-  apt: pkg={{ item }} state=latest
-  with_items:
-    - libffi6
-    - libffi-dev
-
-- name: Check bundler version
-  shell: bundler --version
-  register: version_of_bundler_installed
-  ignore_errors: yes
+  apt:
+    pkg:
+      - libffi6
+      - libffi-dev
+    state: latest
 
 - name: Install Bundler as a system gem
-  command: gem install bundler
-  when: version_of_bundler_installed.stderr != ""
-
-- name: Symlink Rails logs to common area
-  file: src=/srv/{{ app_name }}/shared/log dest=/var/log/rails state=link force=yes
+  gem:
+    name: bundler
+    user_install: no
+    version: '~> 1.17.2'
```

Comment:
Perhaps we should make this a variable?

Reply:
you could, and then set a default. I suppose you could do the same with the Ubuntu version